aiohttp/client.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, *, connector=None, loop=None, cookies=None,
         if loop.get_debug():
             self._source_traceback = traceback.extract_stack(sys._getframe(1))
 
-        if not loop.is_running():
+        if not loop.is_running() and loop is not None:
```

Comment:
uh, so I tested it and sadly the final for loop line must be ``if loop is not None:`` because otherwise if I check if the loop is running with that None check it would raise an ``AttributeError``.